wiki/Client/File_formats/osr_(file_format)/de.md

```diff
@@ -0,0 +1,101 @@
+
+## Format
+
+Byte-Offsets sind in dieser Tabelle nicht enthalten, da die Werte unterschiedlich lang sein können.
+
+| Datentyp | Beschreibung |
+| :-- | :-- |
+| Byte | Spielmodus des Replays (0 = osu!, 1 = osu!taiko, 2 = osu!catch, 3 = osu!mania) |
+| Integer | Version des Spiels, in der das Replay erstellt wurde (z. B. 20131216) |
+| String | MD5-Hash der osu!-Beatmap |
+| String | Spielername |
+| String | MD5-Hash des osu!-Replays (umfasst bestimmte Angaben des Replays) |
+| Short | Anzahl der 300er |
+| Short | Anzahl der 100er in osu!, 150er in osu!taiko, 100er in osu!catch, 100er in osu!mania |
+| Short | Anzahl der 50er in osu!, kleine Fruits in osu!catch, 50er in osu!mania |
+| Short | Anzahl der Gekis in osu!, Max 300er in osu!mania |
+| Short | Anzahl der Katus in osu!, 200er in osu!mania |
+| Short | Anzahl der Misses |
+| Integer | Gesamtpunktzahl (wird auf der Ergebnisanzeige dargestellt) |
+| Short | Höchste Combo (wird auf der Ergebnisanzeige dargestellt) |
+| Byte | Perfect/Full Combo (1 = keine Misses und keine Sliderbreaks sowie keine zu früh beendeten Slider) |
+| Integer | Verwendete Mods. Siehe unten für eine Liste der Modwerte. |
+| String | Lebensleiste: Komma-separierte Paare u/v, wobei u die Zeit in Millisekunden nach Beginn des Songs ist und v eine Gleitkommazahl zwischen 0 und 1 ist, die die Menge an übriggebliebenem Leben zu diesem Zeitpunkt darstellt (0 = Lebensleiste ist leer, 1 = Lebensleiste ist gefüllt) |
+| Long | Zeitstempel ([Windows Ticks](https://learn.microsoft.com/en-us/dotnet/api/system.datetime.ticks)) |
+| Integer | Länge in Bytes der komprimierten Replay-Daten |
+| Byte Array | Komprimierte Replay-Daten |
+| Long | Online-ID des Scores |
+| Double | Zusätzliche Mod-Informationen. Nur vorhanden, wenn [Target Practice](/wiki/Gameplay/Game_modifier/Target_Practice) aktiviert ist. |
+
+**Zusätzliche Mod-Informationen:**
+
+| Mod | Gespeicherte Informationen |
+| :-- | :-- |
+| Target Practice | Gesamtgenauigkeit aller Treffer. Dies durch die Anzahl an Objekten in der Beatmap zu teilen ergibt die Genauigkeit, die im Spiel angezeigt wird. |
+
+Die restlichen Daten enthalten Informationen über die Mausbewegung und die gedrückten Tasten als [LZMA](https://de.wikipedia.org/wiki/Lempel-Ziv-Markow-Algorithmus)-Kodierung.
+
+In der dekomprimierten Form sind die Daten mit Kommas getrennt. Jeder Teil kennzeichnet eine Aktion, die von 4 Zahlen in der Form `w | x | y | z` repräsentiert werden:
+
+| Buchstabe | Datentyp | Beschreibung |
+| :-- | :-- | :-- |
+| w | Long | Zeit in Millisekunden seit der letzten Aktion |
+| x | Float | X-Koordinate des Cursors von 0 bis 512 |
+| y | Float | Y-Koordinate des Cursors von 0 bis 384 |
+| z | Integer | Bitweise Kombination der gedrückten Tasten auf der Tastatur oder der Maus (M1 = 1, M2 = 2, K1 = 4, K2 = 8, Smoke = 16) |
```

Comment:
I removed the explanation here at the end because it was more confusing at first than it should have been. Though, this should be clear from the context alone (binary system).